sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/AddressResolverTest.java

```diff
@@ -283,7 +283,7 @@ private void TestCacheRefreshWhileRouteByPartitionKeyRangeId(
         try {
             resolvedAddresses = this.addressResolver.resolveAsync(request, forceAddressRefresh).block();
         } catch (RuntimeException e) {
-            throw (Exception) e.getCause();
+            throw e;
```

Comment:
same here it is a logic change not code style

Reply:
Reverted